### PR TITLE
Grammatically update the doc

### PR DESCRIPTION
**Pull request by mohamed-khoualed**
Small grammatical update to the  Bootstrap doc

### CODE REVIEW

File: site/content/docs/5.0/getting-started/introduction.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Be sure to have your pages set up with the latest design and development standar
 </html>
 ```
 
-That's all you need for overall page requirements. Visit the [Layout docs]({{< docsref "/layout/grid" >}}) or [our official examples]({{< docsref "/examples" >}}) to start laying out your site's content and components.
+That's all you need for the overall page requirements. Visit the [Layout docs]({{< docsref "/layout/grid" >}}) or [our official examples]({{< docsref "/examples" >}}) to start laying out your site's content and components.
```
Comment (patrickhlauke) [inline site/content/docs/5.0/getting-started/introduction.md:104]: both with and without the "the" works (or rather, the sentence is a tiny bit awkward either way). no strong preference either way.

File: site/content/docs/5.0/getting-started/introduction.md [hunk 2]
```diff
@@ -120,7 +120,7 @@ Bootstrap requires the use of the HTML5 doctype. Without it, you'll see some fun
 
 ### Responsive meta tag
 
-Bootstrap is developed *mobile first*, a strategy in which we optimize code for mobile devices first and then scale up components as necessary using CSS media queries. To ensure proper rendering and touch zooming for all devices, **add the responsive viewport meta tag** to your `<head>`.
+Bootstrap is developed *mobile-first*, a strategy in which we optimize code for mobile devices first and then scale up components as necessary using CSS media queries. To ensure proper rendering and touch zooming for all devices, **add the responsive viewport meta tag** to your `<head>`.
```
Comment (patrickhlauke) [inline site/content/docs/5.0/getting-started/introduction.md:123]: in this case, both with and without the hyphen works, I think - and there's quite a few instances of it being used without. Slight preference for having it without, here, as it's not directly an adjective (i.e. if the sentence was "is developed with a mobile-first approach" then yes, i'd go with hyphen ... but in this case it looks better to me without)

File: site/content/docs/5.0/getting-started/introduction.md [hunk 3]
```diff
@@ -130,7 +130,7 @@ You can see an example of this in action in the [starter template](#starter-temp
 
 ### Box-sizing
 
-For more straightforward sizing in CSS, we switch the global `box-sizing` value from `content-box` to `border-box`. This ensures `padding` does not affect the final computed width of an element, but it can cause problems with some third party software like Google Maps and Google Custom Search Engine.
+For more straightforward sizing in CSS, we switch the global `box-sizing` value from `content-box` to `border-box`. This ensures `padding` does not affect the final computed width of an element, but it can cause problems with some third-party software like Google Maps and Google Custom Search Engine.
```
Comment (patrickhlauke) [inline site/content/docs/5.0/getting-started/introduction.md:133]: yup, as with my comment about "mobile first" vs "mobile-first", this is a case where the hyphen does make sense as "third-party" is used as an adjective.

File: site/content/docs/5.0/getting-started/introduction.md [hunk 4]
```diff
@@ -156,6 +156,6 @@ Stay up to date on the development of Bootstrap and reach out to the community w
 - Join [the official Slack room]({{< param slack >}}).
 - Chat with fellow Bootstrappers in IRC. On the `irc.freenode.net` server, in the `##bootstrap` channel.
 - Implementation help may be found at Stack Overflow (tagged [`bootstrap-5`](https://stackoverflow.com/questions/tagged/bootstrap-5)).
-- Developers should use the keyword `bootstrap` on packages which modify or add to the functionality of Bootstrap when distributing through [npm](https://www.npmjs.com/search?q=keywords:bootstrap) or similar delivery mechanisms for maximum discoverability.
+- Developers should use the keyword `bootstrap` on packages that modify or add to the functionality of Bootstrap when distributing through [npm](https://www.npmjs.com/search?q=keywords:bootstrap) or similar delivery mechanisms for maximum discoverability.
```
Comment (patrickhlauke) [inline site/content/docs/5.0/getting-started/introduction.md:159]: both "which" and "that" are valid. using "that" is probably a bit more natural-sounding

**Review by patrickhlauke — DISMISSED** [review]
only change i'd say is leave the "mobile first" without hyphen

File: site/content/docs/5.0/getting-started/introduction.md (hunk 1 above)
Comment (mdo) [inline site/content/docs/5.0/getting-started/introduction.md:104]:
```suggestion
For next steps, visit the [Layout docs]({{< docsref "/layout/grid" >}}) or [our official examples]({{< docsref "/examples" >}}) to start laying out your site's content and components.
```

File: site/content/docs/5.0/getting-started/introduction.md (hunk 2 above)
Comment (mdo) [inline site/content/docs/5.0/getting-started/introduction.md:123]:
```suggestion
Bootstrap is developed *mobile first*, a strategy in which we optimize code for mobile devices first and then scale up components as necessary using CSS media queries. To ensure proper rendering and touch zooming for all devices, **add the responsive viewport meta tag** to your `<head>`.
```

**Review by mdo — APPROVED** [review]

**Pull request merged**